stroeer/core/v1/core_article_service.proto

```diff
@@ -10,10 +10,42 @@ option java_package = "de.stroeer.core.v1";
 
 // Service to manage core articles
 service ArticleService {
-  // Returns a core article by it's id
+  // Returns a core article by it's `id`
   rpc GetArticle (GetArticleRequest) returns (stroeer.core.v1.Article) {}
+
+  // Returns pages of articles by their `home_section`
+  rpc ListArticles (ListArticlesRequest) returns (ListArticlesResponse) {}
+  // also as continuous stream
+  rpc LotsOfArticles (ListArticlesRequest) returns (stream ListArticlesResponse) {}
```

Comment:
this `stream` is going to be interesting o__O

Reply:
Do we want to provide this, yet?